cfg/config.yaml

```diff
@@ -278,11 +279,13 @@ version_mapping:
   "1.24": "cis-1.24"
   "1.25": "cis-1.7"
   "1.26": "cis-1.8"
+  "oke-1.26": "oke-1.26"
```

Comment:
it seems `oke-1.26` doesn't relevant GKE-1.6.0, right?
or did i miss something?